sys/ztimer/auto_init.c

```diff
@@ -98,6 +100,27 @@ ztimer_clock_t *const ZTIMER_MSEC_BASE = &_ztimer_periph_timer_usec.super;
 #  endif
 #endif
 
+#if MODULE_ZTIMER_SEC
+#  if MODULE_ZTIMER_PERIPH_RTC
+static ztimer_periph_rtc_t _ztimer_periph_timer_rtc_sec;
+ztimer_clock_t *const ZTIMER_SEC = &_ztimer_periph_timer_rtc_sec;
+#  elif MODULE_ZTIMER_MSEC
+static ztimer_convert_frac_t _ztimer_convert_frac_sec;
+ztimer_clock_t *const ZTIMER_SEC = &_ztimer_convert_frac_sec.super.super;
+ztimer_clock_t *const ZTIMER_SEC_BASE = &_ztimer_periph_timer_msec.super;
+#    define ZTIMER_SEC_CONVERT_LOWER        (&_ztimer_periph_timer_rtt_msec)
```

Comment:
is it ok to use this symbol here? doesn't it need the rtt?

Reply:
Good catch. I changed the preprocessor conditional to only uses this if both `ztimer_msec` and `ztimer_periph_rtt` are used.

Maybe it makes sense to decouple this from `ztimer_msec` to also profit from the power savings when `ztimer_sec` but not `ztimer_msec` is used. I'll fix this.

Reply:
Done. RTT is now used for `ZTIMER_SEC` when available regardless of `ZTIMER_MSEC`, unless the application/user explicitly pulled in `ztimer_periph_rtc`.